pkg/webhook/policy.go

```diff
@@ -39,8 +39,8 @@ var (
 	codecs                             = serializer.NewCodecFactory(runtimeScheme)
 	deserializer                       = codecs.UniversalDeserializer()
 	disableEnforcementActionValidation = flag.Bool("disable-enforcementaction-validation", false, "disable validation of the enforcementAction field of a constraint")
-	webhookName                        = flag.String("webhook-name", "validation.gatekeeper.sh", "DEPRECATED: set this on the manifest YAML if needed")
```

Comment:
removing webhookname flag altogether is a breaking change, though there are other changes, so may be fine

Reply:
We should make the webhook name customizable and this should be the flag that allows users to provide that.

Reply:
if we keep it, we'll have to add `// nolint:varcheck,deadcode,unused` to ignore linting

Reply:
@ritazh Webhook name (as in validation.gatekeeper.sh) is fully customizable via VWH config,

Reply:
but I dont think users can deploy VWH separately right?

Reply:
or did you mean they can modify this value in the deploy.yaml: 

https://github.com/open-policy-agent/gatekeeper/blob/master/deploy/gatekeeper_kubebuilder_v2.yaml#L358

If so, then I guess we can just remove this flag.

Reply:
Ya, the next time we release, that kubebuilder_v2 yaml will be the order of the day. We should probably have a changeover plan.